scripts/install-conda.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+set -xe
+# Requires environment CONDA_SHA256, CONDA_VERSION, and DEFAULT_ENV
+wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-$CONDA_VERSION-Linux-x86_64.sh
+
+echo "${CONDA_SHA256}  Miniconda3-$CONDA_VERSION-Linux-x86_64.sh" > miniconda.checksum
+
+if [ $(sha256sum -c miniconda.checksum | awk '{print $2}') != "OK" ]; then
+   echo Error when testing checksum
+   exit 1;
+fi
+
+mv Miniconda3-$CONDA_VERSION-Linux-x86_64.sh miniconda.sh
+sh ./miniconda.sh -b -p /opt/conda
+rm miniconda.sh miniconda.checksum
+
+# install mamba and clean up
+conda install -y -c conda-forge mamba
+mamba clean -afy
+find /opt/conda/ -follow -type f -name '*.a' -delete
+find /opt/conda/ -follow -type f -name '*.js.map' -delete
+
+ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh
```

Comment:
🤔  - instead of installing miniforge then installing mamba would it not make sense to install mambaforge then remove these additional steps?

Reply:
That makes sense. Miniforge is currently being downloaded from repo.anaconda however we might need to download [mambaforge from github](https://github.com/conda-forge/miniforge#mambaforge).

I will open an issue to track this for now.